Gemfile

```diff
@@ -11,10 +11,14 @@ gem 'sprockets-rails'
 gem 'stimulus-rails'
 gem 'turbo-rails'
 gem 'tzinfo-data', platforms: %i[windows jruby]
+gem 'bcrypt', '~> 3.1.7'
 
 group :development, :test do
   gem 'dotenv-rails'
   gem 'rspec-rails'
+  gem 'factory_bot_rails', '~> 6.2'
+  gem 'ffaker', '~> 2.21'
+  gem 'shoulda-matchers', '~> 6.2'
```

Comment:
Do we need shoulda-machers in `development`?

Reply:
No Bhai, we can move it into test block.